ui/core/components/inputs/buffs_debuffs.ts

```diff
@@ -0,0 +1,570 @@
+		stats: [Stat.StatArmor]
+	},
+	{
+		config: BlessingOfWisdom,
+		picker: IconPicker,
+		stats: [Stat.StatMP5]
+	},
+	{
+		config: ManaSpringTotem,
+		picker: IconPicker,
+		stats: [Stat.StatMP5]
+	},
+
+	// Misc Buffs
+	{
+		config: Thorns,
+		picker: IconPicker,
+		stats: [Stat.StatArmor]
+	},
+	{
+		config: RetributionAura,
+		picker: IconPicker,
+		stats: [Stat.StatArmor]
+	},
+	{
+		config: Innervate,
+		picker: IconPicker,
+		stats: [Stat.StatMP5]
+	},
+	{
+		config: PowerInfusion,
+		picker: IconPicker,
+		stats: [Stat.StatMP5, Stat.StatSpellPower]
+	},
+] as PickerStatOptions[]
+
+export const WORLD_BUFFS_CONFIG = [
+	{
+		config: BoonOfBlackfathom,
+		picker: IconPicker,
+		stats: [
+			Stat.StatMeleeCrit,
+			// TODO: Stat.StatRangedCrit,
+			Stat.StatSpellCrit,
+			Stat.StatAttackPower
+		]
+	},
+	{
+		config: FengusFerocity,
+		picker: IconPicker,
+		stats: [Stat.StatAttackPower]
+	},
+	{
+		config: MoldarsMoxie,
+		picker: IconPicker,
+		stats: [Stat.StatStamina]
+	},
+	{
+		config: RallyingCryOfTheDragonslayer,
+		picker: IconPicker,
+		stats: [
+			Stat.StatMeleeCrit,
+			// TODO: Stat.StatRangedCrit,
+			Stat.StatSpellCrit,
+			Stat.StatAttackPower,
+		]
+	},
+	{
+		config: SaygesDarkFortune,
+		picker: IconEnumPicker,
+		stats: [],
+	},
+	{
+		config: SongflowerSerenade,
+		picker: IconPicker,
+		stats: []
+	},
+	{
+		config: SpiritOfZandalar,
+		picker: IconPicker,
+		stats: []
+	},
+	{
+		config: WarchiefsBlessing,
+		picker: IconPicker,
+		stats: [
+			Stat.StatHealth,
+			Stat.StatMeleeHaste,
+			Stat.StatMP5,
+		]
+	},
+] as PickerStatOptions[];
+
+export const DEBUFFS_CONFIG = [
+	// Standard Debuffs
+	{ 
+		config: MajorArmorDebuff,
+		stats: [Stat.StatArmorPenetration],
+		picker: MultiIconPicker,
+	},
+	{ 
+		config: CurseOfRecklessness,
+		picker: IconPicker,
+		stats: [Stat.StatAttackPower]
+	},
+	{ 
+		config: FaerieFire,
+		picker: IconPicker,
+		stats: [Stat.StatAttackPower]
+	},
+	// { 
+	// 	config: MinorArmorDebuff,
+	// picker: MultiIconPicker,
+	// 	stats: [Stat.StatArmorPenetration]
+	// },
+	{ 
+		config: BleedDebuff,
+		picker: MultiIconPicker,
+		stats: [Stat.StatAttackPower, Stat.StatRangedAttackPower]
+	},
+	{ 
+		config: SpellISBDebuff,
+		picker: IconPicker,
+		stats: [Stat.StatShadowPower]
+	},
+	{ 
+		config: SpellScorchDebuff,
+		picker: IconPicker,
+		stats: [Stat.StatFirePower]
+	},
+	{ 
+		config: SpellWintersChillDebuff,
+		picker: IconPicker,
+		stats: [Stat.StatFrostPower]
+	},
+	{ 
+		config: AttackPowerDebuff,
+		picker: MultiIconPicker,
+		stats: [Stat.StatArmor]
+	},
+	{ 
+		config: MeleeAttackSpeedDebuff,
+		picker: IconPicker,
+		stats: [Stat.StatArmor]
+	},
+	{ 
+		config: MeleeHitDebuff,
+		picker: IconPicker,
+		stats: [Stat.StatDodge]
+	},
+
+	// Other Debuffs
+	{
+		config: JudgementOfWisdom,
+		picker: IconPicker,
+		stats: [Stat.StatMP5, Stat.StatIntellect],
+	},
+	{
+		config: HuntersMark,
+		picker: IconPicker,
+		stats: [Stat.StatRangedAttackPower],
+	},
+
+	// Misc Debuffs
```

Comment:
Missing the Curse of Vulnerability and Ancient Corrosive Poison debuffs that I had previously added